mod/wikirate_companies/set/type/company/export.rb

```diff
@@ -130,3 +72,58 @@ def identifier_cards names
     field_nest :answer, view: :titled
   end
 end
+
+format :jsonld do
+  view :molecule do
+    company_jsonld(atom)
+  end
+
+  private
+
+  def company_jsonld(a)
+    {
+      "@context" => "#{request.base_url}/context/#{card.type}.jsonld",
+      "@id"      => path(mark: card.name, format: nil),
+      "@type"    => card.type,
+
+      "name"     => a[:name] || a["name"],
+      "alias"    => a[:alias] || a["alias"],
+      "logo"     => a[:image] || a["image"],
+      "country"  => get_country,
+
+      # identifiers (map 1:1 from atom)
+      "lei"                         => a[:legal_entity_identifier] || a["legal_entity_identifier"],
```

Comment:
I don't like this. the code isn't supposed to know anything about  most of these company identifiers; they're all data with no codename. Everything will break if we change a cardname.

I think the solution should be to make it so that you can specify in each company identifier card (a) whether to include it in jsonld and (b) how to create the links. If this is too much for now, we can make a ticket it for it, but we shouldn't put it off too long.